yafsrc/YetAnotherForum.NET/Controls/EditUsersSuspend.ascx.cs

```diff
@@ -127,9 +127,17 @@ protected void Page_Load([NotNull] object sender, [NotNull] EventArgs e)
             }
 
             // add items to the dropdown
-            this.SuspendUnit.Items.Add(new ListItem(this.GetText("PROFILE", "DAYS"), "1"));
-            this.SuspendUnit.Items.Add(new ListItem(this.GetText("PROFILE", "HOURS"), "2"));
-            this.SuspendUnit.Items.Add(new ListItem(this.GetText("PROFILE", "MINUTES"), "3"));
+            ListItem Item = new ListItem();
```

Comment:
This will not work and will cause viewstate issues! you can not re-use list items

Reply:
This works ;) Anyway please accept the changes (or restore that file to the original one).
I suggest to use the original:

```
            this.SuspendUnit.Items.Add(new ListItem(this.GetText("PROFILE", "DAYS"), "1"));
            this.SuspendUnit.Items.Add(new ListItem(this.GetText("PROFILE", "HOURS"), "2"));
            this.SuspendUnit.Items.Add(new ListItem(this.GetText("PROFILE", "MINUTES"), "3"));

```
and add this: 

```
            foreach(ListItem listItem in this.SuspendUnit.Items)
                listItem.Attributes.Add("class", "pr-2");

```